swagger/components/schemas/processorTransformationRequest.yaml

```diff
@@ -0,0 +1,15 @@
+title: Processor Transformation Request
+type: object
+properties:
+  request:
+    type: object
```

Comment:
This is not needed its dynamically added in Controller

Reply:
Removed.

Reply:
Should we remove "request" property from `routerTransfromationRequest.yaml` as well